correlation_formulas/formula_package.json

```diff
@@ -0,0 +1,148 @@
+{	
+	"formulaPackage": {
+		"name": "CrossCorrelations",
+		"creatorName": "Alberto Rivas",
+		"creatorContactInfo": "applied.research@seeq.com"
```

Comment:
I think we have to remove this email. It is or will be disabled soon

Reply:
Just generically support@seeq.com then?

Reply:
yes